config/application.rb

```diff
@@ -28,5 +28,14 @@ class Application < Rails::Application
 
     # Autoloadable lib directories with classes and modules.
     config.autoload_paths += %W(#{config.root}/lib)
+
+    # Spristupnenie konstant
+    config.before_configuration do
+      env_file = File.join(Rails.root, 'config', 'local_env.yml')
+      YAML.load(File.open(env_file)).each do |key, value|
```

Comment:
1. Nesiel by som cez Figaro, nakolko ako pisu na README stranke _Configuration values often include sensitive information. Figaro strives to be secure by default by encouraging a convention that keeps configuration out of Git._. Toto nema s bezpecnostou nic spolocne.

Suhlasim s bodom 2. Nesiel by som cez ENV. Spravil by som to ako YML, ale s tym, ze by sa nerozlisovalo vobec prostredie.

Na druhej strane, taketo riesenie, co spravila Veronika, je univerzalne. Niekedy sa mozeme rozhodnut, ze nejaka konkretna konstatna bude vhodna iba pre ucely developmentu alebo zase ina bude vhodna pre testovanie (aj ked toto by sa nemalo diat), a ina zase pre produkciu. A hrat sa s example subormi je spravidla neprijemne :)